### PR TITLE
Reference the repository template

### PR DESCRIPTION
**Pull request by pellared**
## Why

Make **goyek** adoption easier.

## What

Reference https://github.com/goyek/template

## Checklist

- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated.
- [x] The code changes follow [Effective Go](https://golang.org/doc/effective_go).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,11 @@ Hello world!
 ok      0.001s
 ```
 
+## Repository Template
+
+Use [goyek/template](https://github.com/goyek/template) to creata a new repository
+or when adopting **goyek** by copying most of its files.
```
Comment (agabrys) [inline README.md:130]:
```suggestion
Use [goyek/template](https://github.com/goyek/template) to create a new repository
or when adopting **goyek** by copying most of its files.
```

**Review by agabrys — CHANGES_REQUESTED** [review]
I found a typo 🙂

File: README.md (hunk 1 above)
Comment (pellared) [reply README.md]: Addressed https://github.com/goyek/goyek/pull/123/commits/50acbe3c4fe69bdabf63e9f203154f9a4be9b744

**Comment by agabrys** [comment]
I found one more thing. TOC is not updated.

**Review by agabrys — APPROVED** [review]

**Pull request merged**